contracts/exchange/Orderbook.sol

```diff
@@ -156,11 +154,11 @@ contract Orderbook is IOrderbook, ManagerBase {
                 amountSpentOnOrder = orders[_orderIds[i]].price * orderAmounts[i];
                 
                 // Check if the transaction is still under the Max Spend
-                if (maxSpend == 0) {
+                if (maxSpend >= amountSpentOnOrder) {
```

Comment:
When you moved it, was it cheaper? I assume maxSpend => amountSpentOnOrder is the most common path which is why you reordered it, is that correct?

Reply:
Correct.

Reply:
sounds good.